### PR TITLE
(DO NOT MERGE YET) test: new tracker importer tests 

### PR DESCRIPTION
**Pull request by vilkg**
This PR adds all existing tracker tests to the new importer. Additionally, more test cases are covered.

### CODE REVIEW

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerImporter_enrollmentsTests.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.hisp.dhis.tracker.importer;
```
Comment (luciano-fiandesio) [inline dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerImporter_enrollmentsTests.java:29]: Perhaps you can move this test to an "enrollment" folder, to be in sync with `tei` and `events`

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerImporter_relationshipsTests.java [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.hisp.dhis.tracker.importer;
```
Comment (luciano-fiandesio) [inline dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerImporter_relationshipsTests.java:29]: Perhaps you can move this test to an "relationships" folder, to be in sync with `tei` and `events`

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerImporter_relationshipsTests.java [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+
+        teis = trackerActions.postAndGetJobReport( new File( "src/test/resources/tracker/importer/teis/teis.json" ) )
+            .validateSuccessfulImport().extractImportedTeis();
+
+        JsonObject eventObject = new FileReaderUtils().read( new File( "src/test/resources/tracker/importer/events/events.json" ) )
+            .replacePropertyValuesWithIds( "event" ).get( JsonObject.class );
+
+        events = trackerActions.postAndGetJobReport( eventObject )
+            .validateSuccessfulImport().extractImportedEvents();
+        //ApiResponse response = eventActions.post( eventObject );
```
Comment (luciano-fiandesio) [inline dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerImporter_relationshipsTests.java:137]: Commented out code

**Review by luciano-fiandesio — APPROVED** [review]
Looks great Gintare, just left few minor comments

**Comment by vilkg** [comment]
I will create a new branch and PR. PR comments solved!

**Pull request closed without merging**